client/src/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+const reducers = (state = [], action) => {
+	switch (action.type) {
```

Comment:
This is on the right track. I don't mind if we have simple placeholder actions here that just return state, but as least pass parameters in and make sure redux is working properly. So just make a simple test action.